src/Parse/Ini.php

```diff
@@ -121,7 +121,7 @@ public function expandProperty(&$array, $key, $value)
             return $array = $value;
         }
 
-        $keys = explode('|', $key);
+        $keys = explode('|', (string) $key);
```

Comment:
Why was this change required?

Reply:
I'm sorry I wasn't clear enough. I don't think changes are required as I mentioned earlier the test failed on my end because I had strict type declaration (rain fork). 
In this example the &key passed from the test fixtures is not always a string and test failed. Maybe this could be a future consideration.

Reply:
I would rather deal with why it isn't a string sometimes instead of explicitly converting and potentially covering up issues.